src/sdg/fixtures/django_celery_beat.json

```diff
@@ -67,6 +67,18 @@
         "timezone": "UTC"
```

Comment:
Can you remove the changes here.

I'm afraid this will cause mails on test and acceptance as well. We should set this manually in the production environment. This also allows us to test if it works beforehand.